src/storage/sftp/storage.c

```diff
@@ -99,7 +293,7 @@ storageSftpLibSsh2SessionFreeResource(THIS_VOID)
     {
         do
         {
-            rc = libssh2_session_disconnect_ex(this->session, SSH_DISCONNECT_BY_APPLICATION, "pgbackrest instance shutdown", "");
+            rc = libssh2_session_disconnect_ex(this->session, SSH_DISCONNECT_BY_APPLICATION, "pgBackRest instance shutdown", "");
```

Comment:
Use PROJECT_NAME define here.

Reply:
Updates in
https://github.com/pgbackrest/pgbackrest/pull/2125/commits/4fd017c80716d71a7ffff1f25379b9da196f7001
https://github.com/pgbackrest/pgbackrest/pull/2125/commits/6811405f150c8f1095c3da31d952f0ddebc58be1
https://github.com/pgbackrest/pgbackrest/pull/2125/commits/a3a82046feab0a915e0c116f867818b86ae42621